lib/HttpSignatureStrategy.js

```diff
@@ -11,7 +11,10 @@ var passport = require('passport');
 var util = require('util');
 var BedrockError = bedrock.util.BedrockError;
 var didio = require('did-io');
+var jsonld = require('jsonld');
 var URL = require('url');
+var request = require('request');
+request = request.defaults({json: true, strictSSL: false});
```

Comment:
strictSSL should be an option or parameterized somehow.  Here's how we did it in bedrock-messages-client: https://github.com/digitalbazaar/bedrock-messages-client/blob/master/lib/client.js#L42